app/src/services/api/apiBaseQuery.ts

```diff
@@ -6,7 +6,6 @@ import { API_URL, CSRF_COOKIE_NAME, OIDC_LOGIN_URL } from "../../constants";
 const baseQuery = fetchBaseQuery({
   baseUrl: API_URL,
   credentials: "include",
-  redirect: "manual",
```

Comment:
What does this mean?

Reply:
it told `fetch` (https://developer.mozilla.org/fr/docs/Web/API/WindowOrWorkerGlobalScope/fetch) that we want to handle redirect manually (not to follow a redirect automatically). I think I put that there in the beginning to handle silent reauth, but it's useless and blocked logout to function properly.